qutebrowser/browser/history.py

```diff
@@ -177,7 +183,6 @@ def __init__(self, database: sql.Database, progress: HistoryProgress,
         # Store the last saved url to avoid duplicate immediate saves.
         self._last_url = None
 
-        self.completion = CompletionHistory(database, parent=self)
```

Comment:
Had to move this to the bottom because otherwise it will try to create indexes on columns it doesn't yet have. Maybe index creation should happen inside `SqlTable.__init__` instead of it being a separate method?

Reply:
Actually, I think we need something like that, otherwise now every time we re-create CompletionHistory without user version changing, its indexes wont be re-created.

Reply:
I added a `force` parameter to `create_index`, with the same behavior as the one in `_create_table`. I think this is enough for now.